src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -4173,12 +4173,12 @@ def create_vnet_gateway(cmd, resource_group_name, virtual_network_gateway_name,
         vnet_gateway.bgp_settings = BgpSettings(asn=asn, bgp_peering_address=bgp_peering_address,
                                                 peer_weight=peer_weight)
 
-    if any((address_prefixes, radius_secret, radius_server, client_protocol)):
+    if any((address_prefixes, client_protocol)):
         vnet_gateway.vpn_client_configuration = VpnClientConfiguration()
         vnet_gateway.vpn_client_configuration.vpn_client_address_pool = AddressSpace()
         vnet_gateway.vpn_client_configuration.vpn_client_address_pool.address_prefixes = address_prefixes
-        if cmd.supported_api_version(min_api='2017-06-01'):
-            vnet_gateway.vpn_client_configuration.vpn_client_protocols = client_protocol
+        vnet_gateway.vpn_client_configuration.vpn_client_protocols = client_protocol
```

Comment:
is this supported in the api version below 2017-06-01?

Reply:
Yes. [here](https://github.com/Azure/azure-rest-api-specs/blob/b1c1a2b9abc9f0194e52f232309e0d3ee0f37545/specification/network/resource-manager/Microsoft.Network/stable/2017-06-01/virtualNetworkGateway.json#L1285)

And vpn client protocol is not strong tied to either RADIUS or AAD.